app-admin/graylog/graylog-3.1.3.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Free and open source log management"
+HOMEPAGE="https://www.graylog.org"
+SRC_URI="https://downloads.graylog.org/releases/graylog/${P}.tgz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
```

Comment:
Thank you both. I noticed this has ppd64 KW but I guess the codebase is similar enough to graylog2 to not make it re-keywordable. Ie, I consider this more of a pkgmove than pure new package.

Tomas, could you open a bug to start discussing about last-riting graylog2? Can you CC me there, I want to see chainsaw's opinion before pushing it forward.

Reply:
Thank you @juippis.

https://bugs.gentoo.org/701728